src/main/java/com/commercetools/sync/commons/helpers/BaseSyncStatistics.java

```diff
@@ -223,15 +224,15 @@ public long getProcessingTimeInMillis() {
      * class.
      *
      * @param statistics the instance of {@link BaseSyncStatistics} from which to create a JSON String.
-     * @return
+     * @return a JSON representation of the given {@code statistics} as a String.
      */
-    public static String getStatisticsAsJSONString(@Nonnull final BaseSyncStatistics statistics) {
+    public static String getStatisticsAsJsonString(@Nonnull final BaseSyncStatistics statistics) {
         String result = null;
         final ObjectMapper mapper = new ObjectMapper();
         try {
             result = mapper.writeValueAsString(statistics);
-        } catch (JsonProcessingException e) {
-            LOGGER.error("Failed to build JSON String of summary.", e);
+        } catch (JsonProcessingException processingException) {
+            LOGGER.error("Failed to build JSON String of summary.", processingException);
         }
         return result;
```

Comment:
Actually, this method was just used for testing. Now I moved it to the testing utils: 	dda9ae3